### PR TITLE
3scale update to 0.7.0

### PR DESCRIPTION
**Pull request by fh-automatron**
Update 3scale manifest to 0.7.0 

This is an automatically generated branch to track the '0.7.0' version of the '3scale' OLM manifests.
This branch will be automatically updated with any changes related to this version until it finally GAs. 

Changes required to fix installation issues should be applied to this branch following the usual [PR Workflow](https://github.com/integr8ly/delorean/blob/master/docs/integrealy-ci/prow-pr-workflow.md)

More details about the Early Warning System that generated this can be found [here](https://github.com/integr8ly/delorean/tree/master/docs/ews)

## Type of change

- [X] Product Update

## Checklist
- [ ] Older product manifests (N-2) have been removed
- [ ] Operator and Product Versions are updated correctly
- [ ] Go Modules are updated if required

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,7 @@ module github.com/integr8ly/integreatly-operator
 go 1.13
 
 require (
-	github.com/3scale/3scale-operator v0.6.0
-	// github.com/3scale/3scale-operator v0.2.1-0.20200730110533-c3b57b704d73
+	github.com/3scale/3scale-operator v0.2.1-0.20201120164439-360b1f382716
```
Comment (laurafitzgerald) [inline go.mod:6]: @trepel should this one be the 0.7.0 tag when it's available?

Comment (trepel) [reply go.mod:6]: @laurafitzgerald good catch. It seems so, yes. I will be checking the availability of the tag and update once it appears.

**Pull request merged**